actionpack/lib/action_controller/metal.rb

```diff
@@ -169,6 +169,8 @@ def controller_name
     # The ActionDispatch::Response instance for the current response.
     attr_internal_reader :response
 
+    ##
+    # Delegates to ActionDispatch::Request#session.
```

Comment:
I think it makes sense to link to the Session guide:
```suggestion
    ##
    # The ActionDispatch::Session instance for the current request.
    # See further details in the
    # {Active Controller Session guide}[https://guides.rubyonrails.org/action_controller_overview.html#session].
```